packages/rollup/src/rollup_bundle.bzl

```diff
@@ -168,6 +168,9 @@ Otherwise, the outputs are assumed to be a single file.
         cfg = "host",
         default = "@npm//rollup/bin:rollup",
     ),
+    "silent": attr.bool(
```

Comment:
Could you add a warning in this doc that using silent can cause rollup to ignore problems in your code, and link to https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#onwarn and maybe https://github.com/rollup/rollup/issues/2686 ?

You could also mention that Bazel expects tools to print nothing on success, so enabling `--silent` gives a more Bazel-idiomatic experience, but the default here is False due to how it also drops warnings.

Reply:
Done, I added this information to the `attr` docstring.